testing-suite/staking-v4/scenarios/PR_6114.py

```diff
@@ -0,0 +1,181 @@
+import threading
+import time
+
+from core.validatorKey import *
+from core.chain_simulator import *
+from chain_commander import *
+from staking import stake, unStake, unBondNodes
+from get_infos.get_validator_info import get_keys_state, get_keys_from_validator_statistics, \
+    get_keys_from_validator_auction
+import requests
+
+
+# SCENARIO 1
```

Comment:
We are missing scenario 2, but it can come later

Reply:
Will be added later with another PR.